registry.go

```diff
@@ -221,11 +221,11 @@ func (r *SchemaRegistry) SubjectVersionsByID(ctx context.Context, id int) ([]sr.
 	return sss, nil
 }
 
-func (r *SchemaRegistry) SchemaVersionsBySubject(ctx context.Context, subject string) ([]sr.SubjectSchema, error) {
+func (r *SchemaRegistry) SchemaVersionsBySubject(ctx context.Context, subject string) ([]int, error) {
 	r.m.Lock()
 	defer r.m.Unlock()
 
-	sss := make([]sr.SubjectSchema, len(r.subjectVersionCache[subject]))
+	sss := make([]int, len(r.subjectVersionCache[subject]))
```

Comment:
Actually, we can now optimize this a lot:
```suggestion
	versions := make([]int, len(r.subjectVersionCache[subject]))
	copy(versions, r.subjectVersionCache[subject])
	return versions
```

Reply:
And if we need a function that returns `[]sr.SchemaSubject` in the future we can create a separate one, like `SchemaSubjectsBySubject`.

Reply:
Good point, just pushed the changes.